### PR TITLE
Canvas

### PR DESCRIPTION
**Pull request by StefanIvanov**

### CODE REVIEW

File: data/tokens.json
Comment (Ddungyov) [inline data/tokens.json:3946]: shouldn't those be inherited by Icon Button's tokens instead of having their own?

File: data/tokens.json
Comment (Ddungyov) [inline data/tokens.json:3962]: Should offset be tokenized? Isn't that part of a larger container that is not part of the component itself?

File: data/tokens.json
Comment (StefanIvanov) [reply data/tokens.json:3946]: we are reworking the icon buttons right now but I would generally say yes, we should have as little overrides for styles as possible

File: data/tokens.json
Comment (StefanIvanov) [reply data/tokens.json:3962]: I see your point and it gives me better idea, but we will have to keep the tokens as such, just use them differently in Figma, thanks!!!

Comment (Ddungyov) [reply data/tokens.json:3962]: Curious to find one 😆

Comment (StefanIvanov) [reply data/tokens.json:3962]: Nothing special, just a frame with paddings that you overlay the canvas with :)

Comment (StefanIvanov) [reply data/tokens.json:3962]: Just pushed that so check out the commit below and the file and resolve this if you like it better now

**Review by Ddungyov — APPROVED** [review]

**Comment by StefanIvanov** [comment]
Will wait to see what reworks we will do for icon buttons before merging this one

**Comment by StefanIvanov** [comment]
@Ddungyov after you approve once more we can merge this

File: data/tokens.json
Comment (Ddungyov) [inline data/tokens.json:822]: I think those should be "gtmd_color-style_color-selected" , just invert the place since it will be more consistent with everywhere else we reuse states.

File: data/tokens.json
Comment (Ddungyov) [inline data/tokens.json:972]: What's the use case for that one?

File: data/tokens.json
Comment (Ddungyov) [inline data/tokens.json:821]: I think those should be "gtmd_color-style_color-selected" , just invert the place since it will be more consistent with everywhere else we reuse states.

File: data/tokens.json
Comment (StefanIvanov) [reply data/tokens.json:972]: these are the opacity values detached from the hover/selected whatever but intended for them just like the color that we apply opacity to

**Review by Ddungyov — APPROVED** [review]
image

**Pull request merged**
